ui/enums/BalanceStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum BalanceStatus {
```

Comment:
Why enums in their own files and directory? Isn't that an overkill?

Reply:
I just didn't want to have the files in the root ui directory.